lib/matplotlib/contour.py

```diff
@@ -1672,7 +1672,9 @@ def _initialize_x_y(self, z):
           contour(Z,N)
           contour(X,Y,Z,N)
 
-        contour up to *N* automatically-chosen levels.
+        contour up to *N+1* automatically-chosen levels.
+        If exactly *N* levels are required either *V* or an
```

Comment:
`*V*` is described right below.  I'd favour either just the first sentence or maybe even:

`contour up to *N+1* automatically chosed contour levels (*N* intervals).`

Just to make it not seem like we are bonkers to have this convention.